build-tools-internal/version.properties

```diff
@@ -23,6 +23,8 @@ netty             = 4.1.76.Final
 
 commons_lang3                   = 3.9
 
+s2_geometry_library_java        = 1.0.1
```

Comment:
I think this can just be local to the legacy-geo module. Versions listed in this file are normally for when they need to be shared across components.